src/utils/apis/TasksApi.js

```diff
@@ -14,7 +15,7 @@ export const tasksApi = {
                     Swal.close() 
                     return response.json()
                 })
-                .catch(err => {
+                .catch(() => {
```

Comment:
Tu nie ma żadnego errora?

Reply:
No jest, ale po co nam on tu?

Reply:
W sensie tutaj @Rados13 chodziło bardziej o to, że nigdzie go nie wykorzystujemy, więc tak naprawdę nie musimy przyjmować go jako argumentu lambdy. W Kotlinie zrobiłbyś `_ -> whatever()`